core/src/test/java/io/questdb/test/TestListener.java

```diff
@@ -90,13 +89,19 @@ public void testFailure(Failure failure) {
 
     @Override
     public void testFinished(Description description) {
-        LOG.infoW().$("<<<< ").$(description.getClassName()).$('.').$(description.getMethodName()).$(" duration_ms=").$(getTestDuration()).$();
+        LOG.infoW().$("<<<< ").$(description.getClassName()).$('.')
+                .utf8(description.getMethodName())
+                .$(" duration_ms=").$(getTestDuration()).$();
+        System.out.println("<<<<= " + description.getClassName() + '.' + description.getMethodName() + " duration_ms=" + getTestDuration());
     }
 
     @Override
     public void testStarted(Description description) {
         testStartMs = System.currentTimeMillis();
-        LOG.infoW().$(">>>> ").$(description.getClassName()).$('.').$(description.getMethodName()).$();
+        LOG.infoW().$(">>>> ").$(description.getClassName()).$('.')
+                .utf8(description.getMethodName())
```

Comment:
`description.getMethodName()` can be invalid utf-8, please log with $invalid(). `ParseNanosFuzzTest` is the example where it will be invalid